README.md

```diff
@@ -1,6 +1,6 @@
 ![](https://img.shields.io/badge/Microverse-blueviolet)
 
-# MY PROFESSIONAL PORTFOLIO
+# REFUGEE FOOD FESTIVAL (Html/css capstone)
```

Comment:
- [ ] Kindly as the previous review asked, add the "Acknowledgements" section or similar that gives appropriate credit to [Cindy Shin](https://www.behance.net/adagio07) the [author of the original design](https://www.behance.net/gallery/29845175/CC-Global-Summit-2015).

Here your section is not mention that
![image](https://user-images.githubusercontent.com/35535130/157994631-f0008520-af03-4a71-aebc-934123c97e1b.png)

Reply:
Addressed ✔